code/mob/living.dm

```diff
@@ -814,7 +826,17 @@
 				VT = "radio"
 				ending = 0
 
-		if (ending == "?")
+		if (singing || (src.bioHolder && src.bioHolder.HasEffect("elvis")))
+			if (src.get_brain_damage() >= 60 || (src.bioHolder && (src.bioHolder.HasEffect("unintelligable") || \
+			src.bioHolder.HasEffect("drunk") || (src.reagents && src.reagents.get_reagent_amount("ethanol") > 40))))
```

Comment:
if they have that much ethanol they should already be drunk I think

Reply:
when I tested and used Add Reagent it didn't succeed, so I wanted to make sure